### PR TITLE
Lab 03

### PR DESCRIPTION
**Pull request by justin98366**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
-else{
+}else{
   alert("Your response wasn't yes or no!");
 }
+
+var number = 7;
+var done = true;
+var rightNum;
+for (var num = 4; num > 0 && done; num--){
+  var rightNum = prompt('What is my favorite number?');
```
Comment (Tenlia) [inline app.js]: You've already declared this variable above the for loop.
Also, rightNum implies that the guess is the right number. You want to show that this isn't the answer but the user imput with the name.

File: app.js [hunk 2]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
-}
-else{
+}else{
   alert("Your response wasn't yes or no!");
 }
+
+var number = 7;
+var done = true;
+var rightNum;
+for (var num = 4; num > 0 && done; num--){
```
Comment (Tenlia) [inline app.js]: does num mean attempts?

File: app.js [hunk 3]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
-else if (people.trim().toLowerCase() == 'no' || people.trim().toLowerCase() == 'n' ){
+}else if (people.trim().toLowerCase() == 'no' || people.trim().toLowerCase() == 'n' ){
   alert('What kind of person do you think I am!?');
-}
-else{
+}else{
   alert("Your response wasn't yes or no!");
 }
+
+var number = 7;
```
Comment (Tenlia) [inline app.js:69]: should this be rightNum?

File: app.js [hunk 4]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
+var done = true;
```
Comment (Tenlia) [inline app.js]: maybe notDone, or set done to false to begin with?

File: app.js [hunk 5]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
+  }else{
+    alert('You\'re too low!');
+  }
+}
+
+var countries = ['afghanistan', 'italy', 'germany']
+var rightCountry;
+var done2 = true;
+for (var place = 0; place < 6 && done2; place++){
+  var rightCountry = prompt('Can you name a country that I lived in?');
```
Comment (Tenlia) [inline app.js]: You declared this variable already

File: app.js [hunk 6]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
+  }else if(rightNum > number){
+    alert('You\'re too high!');
+
+  }else{
+    alert('You\'re too low!');
+  }
+}
+
+var countries = ['afghanistan', 'italy', 'germany']
+var rightCountry;
```
Comment (Tenlia) [inline app.js]: is this the correct answer? or is it the user guess?

File: app.js [hunk 7]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
+var done2 = true;
+for (var place = 0; place < 6 && done2; place++){
```
Comment (Tenlia) [inline app.js]: does place mean tries/attempts?

File: app.js [hunk 8]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
+
+
+
+
+
+
+
+
+
+//
```
Comment (Tenlia) [inline app.js]: what happened here? that's a lot of whitespace

File: style.css [hunk 9]
```diff
@@ -1,10 +1,36 @@
+
+
+
 body {
-  /*background-color: yellow;*/
-  background-image: url("https://pix-media.priceonomics-media.com/blog/1238/generic.jpg");
-  background-color: #cccccc;
+  background-color: black;
+  /*background-image: url("https://pix-media.priceonomics-media.com/blog/1238/generic.jpg");
+  background-color: #cccccc;*/
```
Comment (Tenlia) [inline style.css]: commented-out code should be removed before being turned-in.

File: app.js [hunk 10]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
+}
+
+var tally = 0;
+if(rightCountry === 'italy'){
+     tally++;
+}
+if(rightCountry === 'germany'){
+  tally++;
+}if(rightCountry === 'afghanistan'){
+  tally++;
```
Comment (Tenlia) [inline app.js]: you can use your logical operators to collapse these into one if statement. Should make it more readable, too!

File: app.js [hunk 11]
```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
...
+    tally = tally;
+}else if(juice.trim().toLowerCase() == 'no' || juice.trim().toLowerCase() == 'n'){
+  tally++;
+
+}if(people === null){
+    tally = tally;
+}else if(people.trim().toLowerCase() == 'yes' || people.trim().toLowerCase() == 'y'){
+  tally++;
+}
+alert('You answered ' + tally + ' out of 7 correct, what ever your name is!');
```
Comment (Tenlia) [inline app.js:147]: love it! very creative way of working without the user name

File: app.js (hunk 7 above)
Comment (justin98366) [reply app.js]: yes

File: app.js (hunk 6 above)
Comment (justin98366) [reply app.js]: the user's guess. I changed it to guessCountry

File: app.js (hunk 5 above)
Comment (justin98366) [reply app.js]: fixed

File: app.js (hunk 7 above)
Comment (justin98366) [reply app.js]: i changed it to "i"

File: app.js (hunk 2 above)
Comment (justin98366) [reply app.js]: i changed it to "i"

File: app.js [hunk 12]
```diff
@@ -7,56 +7,148 @@ console.log("var = animals asks the user if I like animals");
...
+
+}
+
+var tally = 0;
+// if(rightCountry === 'italy'){
+//      tally++;
+// }
+// if(rightCountry === 'germany'){
+//   tally++;
+//}
```
Comment (Tenlia) [inline app.js]: be sure to remove your commented-out code!

**Pull request merged**